src/components/Select/Select.tsx

```diff
@@ -370,6 +371,10 @@ export const Select: React.FC<SelectProps> = ({
         return null;
       }
 
+      if (child.type === GroupHeading) {
```

Comment:
I was able to find the `type` attribute in dev forum conversations and looked at the React source behind it. This seems to do what I want, but I didn't see it called out explicitly anywhere in the React docs. Not sure how we feel about that :confused:

Reply:
I think this is okay for now after searching around a bit as well.  I see a few other libraries doing this too.  If React ends up pulling this at some point, hopefully they'll provide a new, documented replacement! 👍